pkg/subctl/cmd/validate_deployment.go

```diff
@@ -20,13 +20,14 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 var validatePodsCmd = &cobra.Command{
-	Use:   "pods",
+	Use:   "deployment",
 	Short: "Validate the submariner pods",
 	Long:  "This command checks that all the submariner pods are running",
```

Comment:
We should probably update the help messages as well:

```suggestion
	Short: "Validate the submariner deployment",
	Long:  "This command checks that the submariner components are properly deployed and running",
```